src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -0,0 +1,131 @@
+  link?: { path: string };
+}
+
+interface PortraitVideoCarouselProps {
+  title: string;
+  items: PortraitVideoItem[];
+}
+
+const PortraitVideoCarousel = ({
+  title,
+  items,
+}: PortraitVideoCarouselProps) => {
+  const { dir } = useContext(ServiceContext);
+  const scrollRef = useRef<HTMLDivElement>(null);
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+
+  const checkScrollButtons = () => {
+    if (!scrollRef.current) return;
+    const { scrollLeft, scrollWidth, clientWidth } = scrollRef.current;
+
+    setCanScrollLeft(scrollLeft > 0);
+    setCanScrollRight(scrollLeft + clientWidth < scrollWidth);
+  };
+
+  const scroll = (direction: 'left' | 'right') => {
+    if (!scrollRef.current) return;
+    const scrollAmount =
+      direction === 'left' ? -PROMO_ITEM_WIDTH : PROMO_ITEM_WIDTH;
+    scrollRef.current.scrollBy({
+      left: scrollAmount,
+      behavior: 'smooth',
+    });
+
+    setTimeout(checkScrollButtons, 100);
+  };
+
+  useEffect(() => {
+    const scrollElement = scrollRef.current;
+    if (scrollElement) {
+      scrollElement.addEventListener('scroll', checkScrollButtons);
+    }
+
+    checkScrollButtons();
+
+    return () => {
+      if (scrollElement) {
+        scrollElement.removeEventListener('scroll', checkScrollButtons);
+      }
+    };
+  }, [items]);
+
+  return (
+    <section
+      aria-label={title}
+      role="region"
+      data-testid="portrait-video-carousel"
+      css={styles.section}
+    >
+      <Heading level={2} size="doublePica" css={styles.heading}>
+        {title}
+      </Heading>
+
+      <div css={styles.scrollContainer}>
+        <div dir={dir} ref={scrollRef} css={styles.scrollWrapper}>
+          {items.map(item => {
+            const image = item.images?.[0]?.url;
+            const alt = item.images?.[0]?.altText || '';
+            const headline = item.headlines?.promoHeadline || '';
+            const href = item.link?.path || '#';
+
+            return (
+              <a key={item.id} href={href} css={styles.promoItem}>
+                {image && (
+                  <img
+                    src={image}
+                    alt={alt}
+                    css={styles.image}
+                    loading="lazy"
+                  />
+                )}
+                <div css={styles.gradientOverlay}>
+                  <Heading
+                    level={3}
+                    size="longPrimer"
+                    css={styles.promoHeading}
+                  >
+                    {headline}
+                  </Heading>
+                </div>
+              </a>
+            );
+          })}
+        </div>
+
+        <div css={styles.buttonGroup}>
+          <button
+            type="button"
+            aria-label="Scroll left"
+            onClick={() => scroll('left')}
+            css={[styles.navButton, !canScrollLeft && styles.disabledButton]}
```

Comment:
The style of the `disabled` button here is invoked depending on the `state` as described above in earlier comment

Reply:
I'd maybe use the `disabled` attribute on the button itself and then in the CSS use the `:disabled` pseudo-class to style, rather than have a CSS class.

Reply:
Cool - i've pushed that now, using the disabled attribute directly from the button now!